lib/tool_belt/cherry_picker.rb

```diff
@@ -27,7 +27,7 @@ def find_cherry_picks(project, release, repo_names)
         commits = issue['changesets']
 
         commits.each do |commit|
-          if !commit['comments'].start_with?('Merge pull request') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
+          if commit['comments'].downcase.start_with?('fixes') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
```

Comment:
@ehelms I didn't realize that some puppet modules don't follow the commit message standard. I changed this logic because NOT 'Merge pull request' would sometimes bring in several commit hashes and only 1 was a valid hash to cherry-pick with; had to manually compare in redmine vs. the log output